commonLibrary/src/main/java/com/skytask/common/Product.java

```diff
@@ -0,0 +1,69 @@
+package com.skytask.common;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Product {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+    private String name;
+    private String description;
+    private double price;
+    private int stock;
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public double getPrice() {
+        return price;
+    }
+
+    public void setPrice(double price) {
+        this.price = price;
+    }
+
+    public int getStock() {
+        return stock;
+    }
+
+    public void setStock(int stock) {
+        this.stock = stock;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
Use Apache utilities for the toString

Reply:
Java 8 StringJoiner is actually better now.

Reply:
@jass-holguin Benchmark link please

Reply:
well the apache StringUtilities was created to provide functionality that Java core libraries did not provide. StringJoiner was created precisely because of this, to provide a way to join Strings with delimiters, prefix and suffix. If there is such a way in the Java core platform we should prefer that, unless there are known problems with it. The code is practically the same. The StringJoiner uses StringBuilder and the implementation is similar to that of the apache one, so...  Anyway, he can use whatever he likes :)